auctions/english-auction/tests/utils/mod.rs

```diff
@@ -0,0 +1,442 @@
+            .call()
+            .await
+            .unwrap()
+    }
+}
+
+pub mod test_helpers {
+
+    use super::*;
+
+    pub async fn create_auction_copy(
+        bid_asset: AuctionAsset,
+        highest_bidder: Option<Identity>,
+        end_block: u64,
+        initial_price: u64,
+        reserve_price: Option<u64>,
+        sell_asset: AuctionAsset,
+        seller: Identity,
+        state: State,
+    ) -> Auction {
+        Auction {
+            bid_asset,
+            highest_bidder,
+            end_block,
+            initial_price,
+            reserve_price,
+            sell_asset,
+            seller,
+            state,
+        }
+    }
+
+    pub async fn defaults_nft() -> (u64, u64, u64, u64, bool) {
+        let sell_count = 1;
+        let inital_count = 1;
+        let reserve_count = 1;
+        let duration = 10;
+        let access_control = true;
+
+        (
+            sell_count,
+            inital_count,
+            reserve_count,
+            duration,
+            access_control,
+        )
+    }
+
+    pub async fn defaults_token() -> (u64, u64, u64, u64) {
+        let sell_amount = 10;
+        let initial_price = 1;
+        let reserve_price = 10;
+        let duration = 10;
+
+        (sell_amount, initial_price, reserve_price, duration)
+    }
+
+    pub async fn nft_asset(contract_id: ContractId, token_id: u64) -> AuctionAsset {
+        let token = NFTAsset {
+            contract_id,
+            token_id,
+        };
+
+        AuctionAsset::NFTAsset(token)
+    }
+
+    pub async fn setup() -> (
+        Metadata,
+        Metadata,
+        Metadata,
+        Metadata,
+        ContractId,
+        ContractId,
+        ContractId,
+        ContractId,
+        ContractId,
+    ) {
+        let num_wallets = 4;
+        let coins_per_wallet = 1;
+        let coin_amount = 1000000;
+        let config = Config {
+            manual_blocks_enabled: true, // Necessary so the `produce_blocks` API can be used locally
+            ..Config::local_node()
+        };
+        let mut wallets = launch_custom_provider_and_get_wallets(
+            WalletsConfig::new(Some(num_wallets), Some(coins_per_wallet), Some(coin_amount)),
+            Some(config),
+        )
+        .await;
+
+        // Get the wallets from that provider
+        let wallet1 = wallets.pop().unwrap();
+        let wallet2 = wallets.pop().unwrap();
+        let wallet3 = wallets.pop().unwrap();
+        let wallet4 = wallets.pop().unwrap();
+
+        let auction_id = Contract::deploy(
+            "./out/debug/english-auction.bin",
+            &wallet1,
+            TxParameters::default(),
+            StorageConfiguration::with_storage_path(Some(
+                "./out/debug/english-auction-storage_slots.json".to_string(),
+            )),
+        )
+        .await
+        .unwrap();
+
+        let sell_asset_id = Contract::deploy(
+            "./tests/artifacts/asset/out/debug/asset.bin",
```

Comment:
It may be beneficial to factor out these hardcoded paths to global variables (or some local module) and use them as variables in case things change in the future. These could be defined somewhere towards the top of the file.

Reply:
check @supiket AMM pr for an example

Reply:
https://github.com/FuelLabs/sway-applications/pull/267/files#diff-c93260d3df375e9c19479a07a4acaf77697d238c68391f46cd23b3a1348d8901